vendor/ezurio/readme.md

```diff
@@ -0,0 +1,937 @@
+        options: one or more of Uplink Options,
+        batteryType: one of Battery Type,
+        sensorReadPeriod: the number of seconds between each sensor update,
+        sensorAggregate: the number of sensor readings per uplink,
+        tempAlarmsEnabled: one of Boolean,
+        tempAlarmLimitLow: the low temperature alarm limit threshold,
+        tempAlarmLimitHigh: the high temperature alarm limit threshold,
+        ledBle: the number of seconds between each flash of the BLE LED,
+        ledLora: the number of seconds between each flash of the LoRa LED
+    }
+
+[Battery Type](#battery-type)
+
+[Uplink Options](#uplink-options)
+
+[Boolean](#boolean)
+
+# Enumerations
+
+Is only one of the following.
+
+## Battery Capacity
+
+|         |
+|---------|
+| 0-5%    |
+| 5-20%   |
+| 20-40%  |
+| 40-60%  |
+| 60-80%  |
+| 80-100% |
+
+## Battery Type
+
+|          |
+|----------|
+| Alkaline |
+| Lithium  |
+
+## Boolean
+
+|       |
+|-------|
+| False |
+| True  |
+
+## Downlink Options
+
+|                         |
+|-------------------------|
+| None                    |
+| Send Simple Config      |
+| Send Advanced Config    |
+| Send Firmware Version   |
+| Send Open Closed Config |
+| Send Battery Voltage    |
+| Reset Sensor            |
+| Start Advertising       |
+| Send RTD Config         |
+
+## Months
+
+|           |
+|-----------|
+| January   |
+| February  |
+| March     |
+| April     |
+| May       |
+| June      |
+| July      |
+| August    |
+| September |
+| October   |
+| November  |
+| December  |
+
+## Open/Closed Sensor State
+
+|        |
+|--------|
+| Closed |
+| Open   |
+
+## Operating Mode
+
+|             |
+|-------------|
+| Door Sensor |
+| Pushbutton  |
+
+# Bitfields
+
+One or more, other than 'None' may be used.
+
+## LoRa Notification Options
+
+|        |
+|--------|
+| None   |
+| Open   |
+| Closed |
+| Resend |
+| Cancel |
+
+## Uplink Message Options
+
+|                                |
+|--------------------------------|
+| None                           |
+| Sensor request for server time |
+| Sensor configuration error     |
+| Sensor alarm flag              |
+| Sensor reset flag              |
+| Sensor fault flag              |
+
+[External RTD Temp Probe Product Brief]: https://www.lairdconnect.com/documentation/product-brief-sentrius-rs1xx-external-rtd-temp-probe/ "External RTD Temp Probe Product Brief"
+[External Temp Sensor Product Brief]: https://www.lairdconnect.com/node/11142/ "External Temp Sensor Product Brief"
+[Integrated Temp & Humidity Sensor Product Brief]: https://www.lairdconnect.com/documentation/product-brief-sentrius-rs1xx-integrated-temp-humidity-sensor/ "Integrated Temp & Humidity Sensor Product Brief"
+[Open/Closed Sensor and Integrated Temp/RH Product Brief]: https://www.lairdconnect.com/documentation/product-brief-sentrius-rs1xx-openclosed-sensor-and-integrated-temprh/ "Open/Closed Sensor and Integrated Temp/RH Product Brief"
```

Comment:
https://www.ezurio.com/documentation/product-brief-sentrius-rs1xx-openclosed-sensor-and-integrated-temprh